compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SpreadMemberNode.java

```diff
@@ -78,7 +78,7 @@ public SpreadMemberNodeModifier modify() {
     /**
      * This is a generated tree node modifier utility.
      *
-     * @since 2201.1.0
+     * @since 2.0.0
```

Comment:
Do we need to commit to this change?

Reply:
Seems to have accidentally added in treegen, I'll revert.